proto/engine/v1/execution_engine.proto

```diff
@@ -60,6 +60,24 @@ message ExecutionPayloadCapella {
 	repeated Withdrawal withdrawals = 15 [(ethereum.eth.ext.ssz_max) = "16"]; // New in Capella.
 }
 
+message ExecutionPayload4844 {
+	bytes parent_hash           = 1 [(ethereum.eth.ext.ssz_size) = "32"];
+	bytes fee_recipient         = 2 [(ethereum.eth.ext.ssz_size) = "20"];
+	bytes state_root            = 3 [(ethereum.eth.ext.ssz_size) = "32"];
+	bytes receipts_root         = 4 [(ethereum.eth.ext.ssz_size) = "32"];
+	bytes logs_bloom            = 5 [(ethereum.eth.ext.ssz_size) = "256"];
+	bytes prev_randao           = 6 [(ethereum.eth.ext.ssz_size) = "32"];
+	uint64 block_number         = 7;
+	uint64 gas_limit            = 8;
+	uint64 gas_used             = 9;
+	uint64 timestamp            = 10;
+	bytes extra_data            = 11 [(ethereum.eth.ext.ssz_max) = "32"];
+	bytes base_fee_per_gas      = 12 [(ethereum.eth.ext.ssz_size) = "32"];
+	bytes block_hash            = 13 [(ethereum.eth.ext.ssz_size) = "32"];
+	repeated bytes transactions = 14 [(ethereum.eth.ext.ssz_size) = "?,?", (ethereum.eth.ext.ssz_max)  = "1048576,1073741824"];
+	uint64 excess_blobs         = 15; // New in EIP-4844
```

Comment:
i'm guessing that, depending on timing of withdrawals vs EIP-4844, we'll either need to merge these two, or add the `repeated Withdrawal withdrawals = 15` field to this. I wonder if we should change the proto field id to `16`

Reply:
I discussed this with @Inphi. I'm keeping solving the Capella + 4844 interaction out of this PR. There has to be some followup, but I don't know what it should be. Likely both should activate with Shanghai, but the Capella implementation in Prysm is very far from complete, so keeping them separated for now.